.github/workflows/cmd-action.yml

```diff
@@ -1,15 +1,45 @@
 name: Command-Action
 
-on: 
+on:
   pull_request:
+  issue_comment:
+    types: [created]
 
 jobs:
   cmd-check:
     runs-on: ubuntu-latest
+    outputs:
+      commands: ${{ steps.cmd.outputs.commands }}
+      branch: ${{ steps.branch.outputs.branch }}
     name: Bot
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4.1.1
+      # Switch to the PR branch if it's triggered by a comment
+      - run: gh pr checkout ${{ github.event.issue.number || github.event.pull_request.number }}
+        if: ${{ github.event.issue.pull_request }}
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
       - uses: paritytech/cmd-action@main
+        id: cmd
         with: 
           commands-directory: '.github/scripts'
           GITHUB_TOKEN: ${{ github.token }}
+      - name: Export branch name
+        run: echo "branch=$(git rev-parse --abbrev-ref HEAD)" >> "$GITHUB_OUTPUT"
+        id: branch
+
+  cmd-run:
+    needs: [cmd-check]
+    if: ${{ github.event.issue.pull_request }}
+    continue-on-error: true
+    # We set the current machine here (to differ between ours and generic ones)
```

Comment:
I don't get this comment, could you elaborate/rephrase?

Reply:
It‘s from the original proposal, so we can define custom self hosted machines for some more demanding jobs.

I will eventually add some more things here, but I would say to ignore it for now